testsuite/drivers/alr.py

```diff
@@ -282,7 +304,8 @@ def index_version():
     return index_branch().split('-')[1]
 
 
-def init_local_crate(name="xxx", binary=True, enter=True, update=True):
+def init_local_crate(name="xxx", binary=True, enter=True, update=True,
+                     with_maintainer_login=False):
```

Comment:
Essentially, yes.

To be more precise, by default (i.e. when pressing &#9166; with a blank input) `alr init` previously added the line `maintainers-logins = ["github-username"]` to the manifest (which is [a real user](https://github.com/github-username) by the way). It now omits the `maintainers-logins` field altogether (since it is no longer required). In both cases, `user.github_login` was/is left unset, so apologies the PR description was somewhat misleading in this respect.

Since some tests run `alr publish` (without `--for-private-index`) on the resulting crate, which fails in the absence of `maintainers-logins`, the `with_maintainer_login` option restores the old behaviour for those tests which need it.